### PR TITLE
fix apicurio project

### PR DESCRIPTION
**Pull request by famartinrh**
server in apicurio project is incorrect and that made application to not deploy.

### CODE REVIEW

File: argocd/overlays/moc-infra/projects/apicurio-registry.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   destinations:
     - namespace: 'apicurio-*'
-      server: 'smaug'
+      server: 'https://api.smaug.na.operate-first.cloud:6443'
```
Comment (HumairAK) [inline argocd/overlays/moc-infra/projects/apicurio-registry.yaml:10]: whoops apologies I missed this last time, for `server` you need to have the url, but if you want to use the _name_ then you have to use the `destination.name` field, like [here](https://github.com/operate-first/apps/blob/master/argocd/overlays/moc-infra/applications/envs/moc/smaug/b4mad/racing.yml#L7)

Comment (famartinrh) [reply argocd/overlays/moc-infra/projects/apicurio-registry.yaml:10]: np, it's fine now

Comment (HumairAK) [reply argocd/overlays/moc-infra/projects/apicurio-registry.yaml:10]: > it's fine now 

Can you switch this to `name` instead? Like so:
```yaml
  destinations:
    - namespace: 'apicurio-*'
      name: 'smaug'
```

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**